doc/release-notes/release-notes-0.6.0.md

```diff
@@ -1,138 +1,49 @@
-private keys from the wallet (dumprivkey, importprivkey).
-
-New JSON-RPC api command for getting information about
-blocks (getblock, getblockhash).
+https://github.com/chaintope/tapyrus-core/issues
 
-New JSON-RPC api command (getmininginfo) for getting
-extra information related to mining. The getinfo
-JSON-RPC command no longer includes mining-related
-information (generate/genproclimit/hashespersec).
+Project source code is hosted at github; you can get
+source-only tarballs/zipballs directly from there:
+https://github.com/chaintope/tapyrus-core/tarball/v0.6.0
 
 
+How to Upgrade
+==============
 
-NOTABLE CHANGES
+It is recommended to upgrade all older nodes to the latest release as it contains fixes to vulnerabilities. If you are running a node on tapyrus testnet follow the instruction in [getting_started](doc/tapyrus/getting_started.md#how-to-start-a-node-on-tapyrus-testnet) to sart a new Tapyrus v0.6.0 node.
 
-BIP30 implemented (security fix for an attack involving
-duplicate "coinbase transactions").
+If you are running a private tapyrus network using versions v0.3.0, v0.4.0 or v0.4.1, v0.5.0, v0.5.1, v0.5.2 shut it down. Wait until it has completely shut down. Follow the instruction in [getting_started](doc/tapyrus/getting_started.md#how-to-start-a-new-tapyrus-network) to sart a new Tapyrus v0.6.0 network. 
 
-The -nolisten, -noupnp and -nodnsseed command-line
-options were renamed to -listen, -upnp and -dnsseed,
-with a default value of 1. The old names are still
-supported for compatibility (so specifying -nolisten
-is automatically interpreted as -listen=0; every
-boolean argument can now be specified as either
--foo or -nofoo).
+Downgrading warning
+-------------------
 
-The -noirc command-line options was renamed to
--irc, with a default value of 0. Run -irc=1 to
-get the old behavior.
+Tapyrus blockchain created by older versions(v0.3.0, v0.4.0 and v0.4.1) is not compatible with v0.6.0 and vice-versa.
 
-Three fill-up-available-memory denial-of-service
-attacks were fixed.
+Compatibility
+==============
 
+Tapyrus v0.6.0 is supported on three platforms, Linux, MacOS and Windows(WSL) in two CPU architectures namely x86_x64 and arm64(aarch64)
 
-NOT YET IMPLEMENTED FEATURES
+0.6.0 change log
+------------------
 
-Support for clicking on bitcoin: URIs and
-opening/launching Bitcoin-Qt is available only on Linux,
-and only if you configure your desktop to launch
-Bitcoin-Qt. All platforms support dragging and dropping
-bitcoin: URIs onto the Bitcoin-Qt window to start
-payment.
+*Dynamic Block size*
 
+This release allows the size of each block in a Tapyrus network to be increased from the default 1MB. This is done with the help of a new xfield type. When the block size is changed using the xfield in a federation block, the new block size becomes the maximum limit for block size for every block created thereafter. Note that in order to sign blocks with the new block size, the signer network needs to be upgraded too. 
```

Comment:
Could you please add a link to a page that explains the new xfield type (`doc/tapyrus/signedblocks.md`)and outline the steps to take when changing the block size?